src/ducks/appSuggestions/AppSuggestion.js

```diff
@@ -27,11 +12,13 @@ class AppSuggestion extends Document {
       _id: transaction._id,
       _type: TRANSACTION_DOCTYPE
     }
-    return pushAtPath(
-      suggestion,
-      'relationships.transactions.data',
-      transactionRelationship
-    )
+
+    // We slice 9 existing transactions because we want to store only 10 items
+    // in the transactions relationship
+    suggestion.relationships.transactions.data = [
+      transactionRelationship,
+      ...suggestion.relationships.transactions.data.slice(0, 9)
```

Comment:
Here the latest suggestion is first. Is it wanted ?

Reply:
Yes it is wanted